app/src/main/java/com/ddoczi/tasky/agenda/presentation/detail/agenda/AgendaDetailEvent.kt

```diff
@@ -0,0 +1,22 @@
+package com.ddoczi.tasky.agenda.presentation.detail.agenda
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+sealed class AgendaDetailEvent {
+    data object OnEdit : AgendaDetailEvent()
+    data object OnSave : AgendaDetailEvent()
+    data object OnClose : AgendaDetailEvent()
+    data object OnDelete : AgendaDetailEvent()
+    data class OnDateSelected(val date: LocalDateTime) : AgendaDetailEvent()
+    data class OnTimeSelected(val time: LocalDateTime) : AgendaDetailEvent()
+    data class OnFromDateSelected(val date: LocalDate) : AgendaDetailEvent()
+    data class OnFromTimeSelected(val time: LocalTime) : AgendaDetailEvent()
+    data object OnNotificationClick : AgendaDetailEvent()
+    data object OnNotificationDismiss : AgendaDetailEvent()
+    data object OnNotificationSelect : AgendaDetailEvent()
```

Comment:
What kind of notification does this refer to? The normal system bar notifications of Android? If yes, these are not really part of this screen, are they?
If you refer to a different type of notification, I think the name `Notification` could be a bit misleading as you notice :D